### PR TITLE
Adaptation to revised ShapeDNA module in LaPy package

### PR DESCRIPTION
**Pull request by kdiers**
This is a partial redesign and rewrite of the brainPrint module, which now includes all external (FreeSurfer) commands that were previously included in the ShapeDNA module within the LaPy package. The post-processing functions (normalization, reweighting, asymmetry) have been moved to the ShapeDNA module. I suggest giving a new tag (0.11 or 0.2) to this version, since it is a major change in usage. The results are identical with the preceding version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,79 +1,61 @@
...
+[BrainPrint-legacy](https://github.com/Deep-MI/BrainPrint-legacy) scripts,
+with the primary goal to provide a Python-only version (except some Freesurfer
+dependencies), to integrate the [LaPy](https://github.com/Deep-MI/LaPy)
+package, and to remove dependencies on third-party software
+(shapeDNA-* binaries, gmsh, meshfix). As a result, some functionality of the
+original BrainPrint-legacy scripts is no longer maintained (currently no
+support of tetrahedral meshes and no support of cortical parcellations or
+label files).
+
+**Note that this branch is the development version that is under active development.**
```
Comment (m-reuter) [inline README.md:13]: Please remove this sentence about development branch. Should not be in the readme.

File: setup.py [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
-    keywords='BrainPrint'
+    python_requires='>=3.4',
+    keywords='BrainPrint',
     # This field lists other packages that your project depends on to run.
     # Any package you put here will be installed by pip when your project is
     # installed, so they must be valid existing projects.
     # For an analysis of "install_requires" vs pip's requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-#    install_requires=['peppercorn'],  # Optional
+    install_requires=['numpy', 'scipy']
```
Comment (m-reuter) [inline setup.py:35]: Once Lapy is added to pip, it should be listed here

File: README.md (hunk 1 above)
Comment (kdiers) [reply README.md]: Done.

File: setup.py (hunk 2 above)
Comment (kdiers) [reply setup.py:35]: OK.

**Pull request merged**